tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -86,29 +85,11 @@ def format_message(self, alert):
         return "Subject: [Alert] %s, %s threshold breached\n\n%s" % (
             alert.resource, alert.severity, alert.tags['message'])
 
-    def get_alert_destinations(self, alert):
```

Comment:
There you are checking the global notification settings like emai_brick_utilization...etc..
as we discussed earlier all users receives all alert then why we need this..